### PR TITLE
Simple logging. 

### PR DESCRIPTION
**Pull request by lemonsaurus**
I don't know if this will show up in the docker logs like Inver wanted, so it might need testing in prod.

### CODE REVIEW

File: pysite/__init__.py [hunk 1]
```diff
@@ -1 +1,32 @@
...
+import logging
+import os
+
+
+# region Logging
+# Get the log level from environment
+log_level = os.environ.get("LOG_LEVEL", "info").lower()
+formatter = logging.Formatter('[%(asctime)s] [%(levelname)s] %(message)s')
+
+if log_level == 'debug':
```
Comment (gdude2002) [inline pysite/__init__.py]: Could do this via hasattr/getattr?

**Review by gdude2002 — CHANGES_REQUESTED** [review]
It might be worth using useful names for specific loggers in specific places, but for now I think this is mostly fine.

**Review by gdude2002 — DISMISSED** [review]

File: pysite/route_manager.py [hunk 2]
```diff
@@ -25,33 +26,34 @@ def __init__(self):
...
 
         for sub in self.subdomains:
             sub_blueprint = Blueprint(sub, __name__, subdomain=sub)
 
-            print(f"Loading Blueprint: {sub_blueprint.name}")
+            self.log.debug(f"Loading Blueprint: {sub_blueprint.name}")
             self.load_views(sub_blueprint, f"pysite/views/{sub}")
             self.app.register_blueprint(sub_blueprint)
-            print("")
+            self.log.debug("")
```
Comment (swedgwood) [inline pysite/route_manager.py]: Why is there an empty print here? (Now self.log.debug)

**Review by swedgwood — COMMENTED** [review]
Looking good to me!

**Review by swedgwood — APPROVED** [review]
👍

**Pull request merged**